texts.js

```diff
@@ -10,5 +10,6 @@ export default {
     'snackbar-saveerror': 'Something went wrong, please try again later',
```

Comment:
Since we'll have breaking changes in the next release, lets rename all these errors to follow the kebab case format, so: `snackbar-save-error`